components-compose/src/main/java/uk/gov/hmrc/components/compose/molecule/input/TextInputView.kt

```diff
@@ -154,68 +103,21 @@ object TextInputView {
                 },
                 prefix = prefix,
                 placeholderText = { placeholderText?.let { Text(text = it) } },
-                supportingText = if (counterEnabled) errorTextCounterCombo else error,
-                trailingIcon = trailingIcon,
+                supportingText = if (counterEnabled)
+                    errorTextCounterCombo(errorText, errorContentDescription, characterCount, localValue)
+                else error(
+                    errorText,
+                    errorContentDescription
+                ),
                 singleLine = singleLine,
                 keyboardOptions = keyboardOptions,
+                visualTransformation = visualTransformation,
+                trailingIcon = trailingIcon ?: clearTrailingIcon,
+                colors = HmrcTheme.textFieldColors,
+                interactionSource = interactionSource
```

Comment:
I don't think we need all these? visual transformation, colors, interaction source don't need to be defined I think... not sure about trailing icon - we don't want to change whatever the existing behaviour was with the clear button.

Reply:
colors was already defined as such, trailingicon I just renamed to make more clear (pun intended!)